python/lsst/ts/m2gui/main_window.py

```diff
@@ -200,26 +206,38 @@ def _get_log_file_name(self):
             pathlib.Path(__file__).parents[0] / ".." / ".." / ".." / ".." / "log" / name
         )
 
-    def _set_model(self, is_simulation_mode):
+    def _set_model(self, is_simulation_mode, is_crio_simulator):
         """Set the model.
 
         Parameters
         ----------
         is_simulation_mode: `bool`
             Is the simulation mode or not.
+        is_crio_simulator : `bool`
+            Is using the M2 cRIO simulator or not.
 
         Returns
         -------
         model : `Model`
             Model object.
+
+        Raises
+        ------
+        `ValueError`
+            When simulation mode and cRIO simulator are used.
         """
 
+        if is_simulation_mode and is_crio_simulator:
```

Comment:
If the argument is an enumeration you don't need to worry about this condition happening.